### ISSUE TITLE
Osu File parser error

### ISSUE THREAD

**Issue by Willian-Zhang** [OSU-Mapper/osu-audio-feature-extract#6]
```
Version:Re:Zero
```
Error:
```
Traceback (most recent call last):
  File "script/osuDiff.py", line 9, in <module>
    searchSublevel("Metadata", "Version")
  File "/Users/willian/Github/OSU-Mapper/osu-audio-feature-extract/script/osuFileParser.py", line 17, in searchSublevel
    l, r = line.split(':')
ValueError: too many values to unpack (expected 2)
```

**Comment by Willian-Zhang** [comment]
![image](https://cloud.githubusercontent.com/assets/5165840/25608903/c545f6b0-2eea-11e7-94cb-574c8fd07d55.png)

**Comment by ly976** [comment]
>>> Extracting: Data/Beatmaps/139677 Demetori - Hoshi no Utsuwa ~ Casket of Star/Casket of Star.mp3
>>>>> Difficualty: Lunatic
>>>>> Difficualty: Normal
>>>>> Difficualty: Yoeri's Easy
>>>>> Difficualty: sjoy's Hard
>>>>> Difficualty: sjoy's Insane
ls: Data/Beatmaps/139677 Demetori - Hoshi no Utsuwa ~ Casket of Star.zip/*.osu: Not a directory
Traceback (most recent call last):
  File "script/audioname.py", line 9, in <module>
    searchSublevel("General", "AudioFilename")
  File "/Users/yanmingjun/Documents/osu-audio-feature-extract/script/osuFileParser.py", line 4, in searchSublevel
    with open(sys.argv[1], 'r') as my_file:
FileNotFoundError: [Errno 2] No such file or directory: ''
>>> Extracting: Data/Beatmaps/139677 Demetori - Hoshi no Utsuwa ~ Casket of Star.zip/
mkdir: Data/Beatmaps/139677 Demetori - Hoshi no Utsuwa ~ Casket of Star.zip/Trainings: Not a directory
Traceback (most recent call last):
  File "script/osuDiff.py", line 9, in <module>
    searchSublevel("Metadata", "Version")
  File "/Users/yanmingjun/Documents/osu-audio-feature-extract/script/osuFileParser.py", line 4, in searchSublevel
    with open(sys.argv[1], 'r') as my_file:
NotADirectoryError: [Errno 20] Not a directory: 'Data/Beatmaps/139677 Demetori - Hoshi no Utsuwa ~ Casket of Star.zip/*.osu'
>>>>> Difficualty: 
>>> Extracting: Data/Beatmaps/162800 Linkin Park - Living Things/lt.mp3

**Comment by Willian-Zhang** [comment]
fixed via 	0fa0276
